projects/WTI-API/packageWTI.xml

```diff
@@ -16,6 +16,10 @@
 	
 	<!-- the directory where the user documentation (user manual) can be found -->
 	<property name="doc.dir" location ="doc/User Manual" />
+	
+	<!-- the set of properties we want to add to the WTI pc2v9.ini file -->
+	<property name="wtiDefaultScoreboardInfo" 
+				value="wtiscoreboardaccount=scoreboard2${line.separator}wtiscoreboardpassword=scoreboard2${line.separator}" />

```

Comment:
In general this is the wrong place to fix a source/configuration file, for a  number of good coding and testing reasons
This change should be done in the .ini file.

Reply:
I think you're missing the point.  This is not about editing the configuration.  This is about establishing the INITIAL (default) configuration.   This line puts an initial value (specifically, "wtiscoreboardaccount=scoreboard2") into the initial default .ini file.  The user can then either use the default, or can edit the .ini and change it.

Failure to include this line (i.e., this change) would mean the user would NEVER be able to start the WTI server without first editing the .ini file and putting a scoreboard account in it.  That would analogous to us having a default pc2v9.ini file which did not contain "server=localhost:50002" in it -- which would mean the user would always have to edit their pc2v9.ini file in order to start the system.  We don't set up pc2v9.ini that way; we put an initial default value in it so the user can execute the pc2 server out of the box with no configuration changes.  This PR change does precisely that same thing for the WTI project -- allows the user to start the WTI server without having to first edit the .ini file (i.e., it establishes a default value for the scoreboard account, while allowing the user to edit/change it if they want).

I request that you review the intent of this change and then that you retract your request for changes to this change.